src/opt.cpp

```diff
@@ -517,7 +521,11 @@ void Optimize(llvm::Module *module, int optLevel) {
 #endif
         optPM.add(llvm::createFunctionInliningPass());
         optPM.add(CreateMakeInternalFuncsStaticPass());
+#if ISPC_LLVM_VERSION <= ISPC_LLVM_11_0
```

Comment:
The same, LLVM 11.1 needs to be handled correctly.
So it needs to be ">= 12.0" and "else".

Reply:
Done